src/store/contacts/contacts-state.ts

```diff
@@ -0,0 +1,16 @@
+import { IAccountBase } from '../../interfaces/account.interface';
+
+export interface ContactsState {
+  contacts: Array<IAccountBase>;
+  blacklistedAddresses: Array<string>;
+  addContactRequest: string;
+}
+export const contactsInitialState: ContactsState = {
+  contacts: [],
+  blacklistedAddresses: [],
+  addContactRequest: ''
+};
+
+export interface ContactsRootState {
+  contacts: ContactsState;
```

Comment:
call this `ContactBookRootState` and change key to `contactBook` to prevent conflict in naming with it's inner `contacts: []`